ote/src/cljs/ote/app/controller/route/route_wizard.cljs

```diff
@@ -285,6 +287,10 @@
   (process-event [{response :response} app]
     (assoc-in app [:route :stops] response))
 
+  LoadRouteOperatorResponse
```

Comment:
While route operator is being loaded, will UI display potentially incorrect data containing whatever was left into appstate by previous visit to this view?
So the sea route operator section should display a loading spinner while operator data is not available
(or alternatively block the whole view)

Reply:
I was adding the spinner but decided to test it first and figured that we don't need the spinner because response is so quick.
But maybe we should add it anyway?

Reply:
I think we should. It's normally fast, but as we've seen in some cases a request may take long using a slow device running low on memory and with corporate sw slowing the browser down. Or with occasional network glitch. When using a loading placeholder, it's guaranteed we don't need to get back to this.
A slow request which blocks user is not as critical, but here user may end up being displayed wrong data.

Reply:
Added two spinner. For the route and for the operator.